### PR TITLE
Add Bandersnatch

### PR DESCRIPTION
**Pull request by oliver-gomes**
Bandersnatch is an extension from Black Mirror and think it deserved a separate spot on it's own

### CODE REVIEW

File: tv_series.md [hunk 1]
```diff
@@ -25,54 +25,66 @@
...
 
 #### Black Mirror
+
 > A television anthology series that shows the dark side of life and technology.
 
 ![Black_Mirror](assets/black_mirror.jpg)
-* Year: 2011-
-* Country: UK
-* [_IMDb_](https://www.imdb.com/title/tt2085059/)
+
```
Comment (greybax) [inline tv_series.md]: too lot of changes.. Why?

Comment (oliver-gomes) [reply tv_series.md]: I just added the Bandersnatch that's all, nothing else changed

Comment (greybax) [reply tv_series.md]: seems you've added spaces after every header.

File: tv_series.md [hunk 2]
```diff
@@ -25,54 +25,66 @@
...
+- Year: 2011-
+- Country: UK
+- [_IMDb_](https://www.imdb.com/title/tt2085059/)
 
 ### 8.6
 
 #### Mr. Robot
+
```
Comment (greybax) [inline tv_series.md]: here..

File: tv_series.md [hunk 3]
```diff
@@ -25,54 +25,66 @@
...
-* Year: 2015-
-* Country: USA
-* [_IMDb_](https://www.imdb.com/title/tt4158110)
+
+- Year: 2015-
+- Country: USA
+- [_IMDb_](https://www.imdb.com/title/tt4158110)
 
 #### The IT Crowd
+
```
Comment (greybax) [inline tv_series.md]: here..

File: tv_series.md [hunk 4]
```diff
@@ -25,54 +25,66 @@
...
-* Year: 2006–2013
-* Country: UK
-* [_IMDb_](https://www.imdb.com/title/tt0487831)
+
+- Year: 2006–2013
+- Country: UK
+- [_IMDb_](https://www.imdb.com/title/tt0487831)
 
 #### Silicon Valley
+
```
Comment (greybax) [inline tv_series.md]: here..

File: tv_series.md [hunk 5]
```diff
@@ -81,44 +93,65 @@
...
-* [IMDb](https://www.imdb.com/title/tt2543312/)
+
+- Year: 2014-2017
+- Country: USA
+- [IMDb](https://www.imdb.com/title/tt2543312/)
 
 ### 8.2
 
 #### Chuck
+
```
Comment (greybax) [inline tv_series.md]: here.. and another places below too

Comment (oliver-gomes) [reply tv_series.md]: Yes, my bad for that my code editor had prettier on and it affected the spacing.

Comment (greybax) [reply tv_series.md]: Could you please fix it?

Comment (oliver-gomes) [reply tv_series.md]: I am closing this then and redo it

**Pull request closed without merging**